cilium-cli.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: cilium-cli
+  version: 0.13.0
+  epoch: 0
+  description: cert-manager CLI tool to manage and configure cert-manager resources

```

Comment:
This should be cilium rather than cert-manager

Reply:
thank you @margamanterola, copy-paste issues 🤦